src/test/scala/io/iohk/ethereum/vm/Generators.scala

```diff
@@ -82,11 +83,29 @@ object Generators extends ObjectGenerators {
       program <- codeGen.map(Program.apply)
       inputData <- inputDataGen
       value <- valueGen
+      blockNumber <- blockNumberGen
+
+      blockHeader = BlockHeader(
+        parentHash = ByteString(Hex.decode("d882d5c210bab4cb7ef0b9f3dc2130cb680959afcd9a8f9bf83ee6f13e2f9da3")),
+        ommersHash = ByteString(Hex.decode("1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347")),
+        beneficiary = ByteString(Hex.decode("95f484419881c6e9b6de7fb3f8ad03763bd49a89")),
+        stateRoot = ByteString(Hex.decode("634a2b20c9e02afdda7157afe384306c5acc4fb9c09b45dc0203c0fbb2fed0e6")),
+        transactionsRoot = ByteString(Hex.decode("56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421")),
+        receiptsRoot = ByteString(Hex.decode("56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421")),
+        logsBloom = ByteString(Hex.decode("00" * 256)),
+        difficulty = BigInt("989772"),
+        number = blockNumber,
+        gasLimit = 131620495,
+        gasUsed = 0,
+        unixTimestamp = 1486752441,
+        extraData = ByteString(Hex.decode("d783010507846765746887676f312e372e33856c696e7578")),
+        mixHash = ByteString(Hex.decode("6bc729364c9b682cfa923ba9480367ebdfa2a9bca2a652fe975e8d5958f696dd")),
+        nonce = ByteString(Hex.decode("797a8f3a494f937b")))
```

Comment:
I would use nulls for the values we don't need. Anyway, I guess it mostly bothers me because it is inside that function. Can we move it outside as `defaultBlockHeader` or something, please?